PySDM/initialisation/aerosol_composition/dry_aerosol.py

```diff
@@ -64,11 +66,10 @@ def volfrac_just_soluble(self, volfrac: dict, soluble=True):
         else:
             _masked = {k: (not self.is_soluble[k]) * volfrac[k] for k in self.compounds}
 
-        _denom = sum(list(_masked.values()))
-        if _denom == 0.0:
-            x = {k: 0.0 for k in self.compounds}
-        else:
-            x = {k: _masked[k] / _denom for k in self.compounds}
+        _denom = np.array(sum(list(_masked.values())))
+        _denom[_denom == 0.0] += 1.0
+
+        x = {k: _masked[k] / _denom for k in self.compounds}
```

Comment:
@slayoo should we also add this change to #922 ?

Reply:
I'm not sure I understand what it does, a unit test would be great - let me ask you for a PR with a unit test, thanks!